### ISSUE TITLE
P1 socket client instead of WebSocket

### ISSUE THREAD

**Issue by jvandijk** [ebaauw/homebridge-p1#61]
Hi Erik,

Thank you for this great plugin. I've been using it for quite a while already using my DSMR 2.2 based setup. But now I need to look into sharing the P1 information with multiple consumers. In my scenario to effectively charge my car.

I changed the configuration to move from the serial interface to the websocket interface. This renders an immediate parse error without receiving any data.

I'm curious why you are using a websocket to communicate to a ser2net based interface, since ser2net does not adhere to the websocket protocol, it's just provides a plain socket. As I see it, there is no handshake involved in setting up the socket to start reading data.

I find it odd, that the current setup does seem to work for DMSR 5 based ser2net setups, while in my case with DSMR 2.2 it errors out on the handshake with: `Parse error, expected HTTP/`

Any thoughts about this? A plain `net.Socket` does seem to work for me.

**Comment by ebaauw** [comment]
> I'm curious why you are using a websocket to communicate to a ser2net based interface

I'm not.  See https://github.com/ebaauw/homebridge-p1/wiki/Multiple-Concurrent-Connections-to-the-Smart-Meter.

**Comment by jvandijk** [comment]
Thank you Erik,

🤦‍♂️ You've solved it way more elegantly than I interpreted. So the WsClient actually only exist as your way of relaying the data over multiple homebridge P1 instances.

Setting the hostname:port connection string in the serialPort attribute immediately works for me.

Maybe it's an option to update the schema.json documentation to add the hostname:port example next to the /dev/ttyUSB0 one. Although your wiki is very clear, I did not understand initially.
